apps/meteor/client/components/message/variants/RoomMessage.tsx

```diff
@@ -62,6 +62,24 @@ const RoomMessage = ({
 
 	const messageRef = useJumpToMessage(message._id);
 
+	// Add a new function for handling double click events
+	const handleRightClick = async (e: React.MouseEvent) => {
+		e.preventDefault();
+
+		//get the button that oppens the menu
+		const moreButton = document.querySelector('[title="More"]');
```

Comment:
I suspect this will only work when your language is English

Reply:
Also, I think you should test where you are clicking, there are links and buttons inside the message where you might want to use the original right click